<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMOGPUListeFormu</class>
 <widget class="QDialog" name="TMOGPUListeFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
TMOGPUListeFormu{
	background-image: url(:/resimler/gpu.jpg);

}

QLabel {
   font:Bold; 
   color: #ffffff;
}
QGroupBox {
    font:Bold;
color: #ffffff;
}
QPushButton{font:Bold;}
QRadioButton{
    font:Bold;
color: #ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="GPAramaBolumu">
     <property name="title">
      <string>GPU ARAMA BÖLÜMÜ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2">
       <widget class="QPushButton" name="PBAra">
        <property name="text">
         <string>ARA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="GPParcaMarkasi">
        <property name="title">
         <string>Parça Markası</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="LPMarkasiDeger">
           <property name="text">
            <string>Değer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LEPMarkasiDeger"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="GPPMarkasiFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBPMarkasiFTIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBPMarkasiFTIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBPMarkasiFTIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="GPParcaModeli">
        <property name="title">
         <string>Parça Modeli</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="LPModeliDeger">
           <property name="text">
            <string>Değer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LEPModeliDeger"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="GPPModeliFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBPModeliFTIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBPModeliIFTIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBPModeliFTIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="PBKapat">
        <property name="text">
         <string>KAPAT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="TWTablo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PBAra</sender>
   <signal>clicked()</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>942</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PBKapat</sender>
   <signal>clicked()</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>890</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>941</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliFTIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>942</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliIFTIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>943</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliFTIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>943</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPModeliDeger</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>943</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPMarkasiDeger</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPModeliDeger</sender>
   <signal>textEdited(QString)</signal>
   <receiver>TMOGPUListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aramaYap()</slot>
 </slots>
</ui>
